### PR TITLE
Fix a bunch of inspection issues

### PR DESCRIPTION
**Pull request by Martmists-GH**

### CODE REVIEW

**Comment by aurieh** [comment]
there is a lot of whitespace(?) diff, did you dos2unix the repository?

File: src/main/kotlin/me/alexflipnote/kawaiibot/utils/RequestUtil.kt [hunk 1]
```diff
@@ -62,7 +62,7 @@ object RequestUtil {
          *
          * @return body as a [kotlin.String]
          */
-        fun string() = isValid { raw.string() }
+        fun string() = this.isValid { raw.string() }!!
```
Comment (aurieh) [inline src/main/kotlin/me/alexflipnote/kawaiibot/utils/RequestUtil.kt]: why? The return type of `isValid` is not nullable.

File: src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt [hunk 2]
```diff
@@ -22,12 +22,12 @@ import net.dv8tion.jda.core.events.message.MessageReceivedEvent
...
  */
 class Arguments internal constructor(private val asIter: StringTokenizer.TokenIterator, private val receivedEvent: MessageReceivedEvent) : Iterator<String> by asIter {
-    val bySpace by lazy { ImmutableList.copyOf(this) }
+    val bySpace: ImmutableList<String> by lazy { ImmutableList.copyOf(this) }
     val asString by lazy { asIter.rest() }
     val asDisplayString by lazy { receivedEvent.jda.resolveNames(bySpace) }
     val asCleanString by lazy { StringUtil.cleanContent(asString) }
     val asCleanerString by lazy { StringUtil.cleanerContent(asString) }
-    val asFlags by lazy { OptParser.parsePosix(bySpace) }
+    private val asFlags by lazy { OptParser.parsePosix(bySpace) }
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt]: this change doesn't seem intentional

File: src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt [hunk 3]
```diff
@@ -38,7 +38,7 @@ class Arguments internal constructor(private val asIter: StringTokenizer.TokenIt
      * @see hasNext
      * @see next
      */
-    fun nextOrNull(): String? {
+    private fun nextOrNull(): String? {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt]: and this

File: src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt [hunk 4]
```diff
@@ -136,7 +136,7 @@ class Arguments internal constructor(private val asIter: StringTokenizer.TokenIt
      * @see BOOLEAN_TRUE
      * @see BOOLEAN_FALSE
      */
-    fun nextBoolean(): Boolean {
+    private fun nextBoolean(): Boolean {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt]: and this

File: src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt [hunk 5]
```diff
@@ -177,7 +177,7 @@ class Arguments internal constructor(private val asIter: StringTokenizer.TokenIt
      * @see userMentionPattern
      * @see userDiscrimPattern
      */
-    fun nextMember(): Member {
+    private fun nextMember(): Member {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt]: and this

File: src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt [hunk 6]
```diff
@@ -260,7 +260,7 @@ class Arguments internal constructor(private val asIter: StringTokenizer.TokenIt
      *
      * @see userMentionPattern
      */
-    fun nextUser(): User {
+    private fun nextUser(): User {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/Arguments.kt]: and this

File: src/main/kotlin/me/aurieh/ichigo/core/CommandHandler.kt [hunk 7]
```diff
@@ -128,7 +126,7 @@ class CommandHandler private constructor(val commands: Map<String, CommandWrappe
         private val checks = HashSet<ICheck>()
         private val developers = HashSet<Long>()
 
-        fun addCommand(commandWrapper: CommandWrapper): Builder {
+        private fun addCommand(commandWrapper: CommandWrapper): Builder {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/CommandHandler.kt]: and this

File: src/main/kotlin/me/aurieh/ichigo/core/CommandHandler.kt [hunk 8]
```diff
@@ -148,7 +146,7 @@ class CommandHandler private constructor(val commands: Map<String, CommandWrappe
             return this
         }
 
-        fun addAlias(alias: String, to: String): Builder {
+        private fun addAlias(alias: String, to: String): Builder {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/core/CommandHandler.kt]: and this

File: src/main/kotlin/me/aurieh/ichigo/utils/StringTokenizer.kt [hunk 9]
```diff
@@ -62,14 +62,14 @@ object StringTokenizer {
...
-    fun tCharCharHashMapOf(vararg pairs: Pair<Char, Char>): TCharCharHashMap {
+    private fun tCharCharHashMapOf(vararg pairs: Pair<Char, Char>): TCharCharHashMap {
         val map = TCharCharHashMap()
         for ((k, v) in pairs)
             map.put(k, v)
         return map
     }
 
-    val defaultQuoteChars = tCharCharHashMapOf(
+    private val defaultQuoteChars = tCharCharHashMapOf(
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/utils/StringTokenizer.kt]: not meant to be private

File: src/main/kotlin/me/aurieh/ichigo/utils/StringTokenizer.kt [hunk 10]
```diff
@@ -62,14 +62,14 @@ object StringTokenizer {
 
     class UnclosedQuoteError(s: String) : Throwable(s)
 
-    fun tCharCharHashMapOf(vararg pairs: Pair<Char, Char>): TCharCharHashMap {
+    private fun tCharCharHashMapOf(vararg pairs: Pair<Char, Char>): TCharCharHashMap {
```
Comment (aurieh) [inline src/main/kotlin/me/aurieh/ichigo/utils/StringTokenizer.kt]: not meant to be private

**Comment by AlexFlipnote** [comment]
damn there's a lot of changes that needs to be made

**Comment by martmists** [comment]
PR was intended for the weebsh thing but I thought I might as well fix whitespace; Inconsistent \r\n and \n in the repo
private: blame intellij, not intended
nullability: blame intellij, not intended

**Comment by martmists** [comment]
This PR now only fixes line separators and some misc issues.

**Comment by martmists** [comment]
@aurieh update on this?

**Comment by aurieh** [comment]
@martmists I will review in a bit

**Pull request merged**